src/applications/static-pages/homepage/HomepageSearch.jsx

```diff
@@ -73,6 +73,24 @@ const HomepageSearch = () => {
       e.target.value,
     )}&t=${false}`;
 
+    // Record the analytic event.
+    recordEvent({
+      event: 'view_search_results',
+      action: 'Homepage - Search',
+      'search-page-path': searchUrl,
+      'search-query': e.target.value,
+      'search-results-total-count': null,
+      'search-results-total-pages': null,
+      'search-selection': 'All VA.gov',
+      'search-typeahead-enabled': false,
```

Comment:
I should have clarified that I also saw this in the header search as well. It seems odd to me, since typeahead _is_ enabled. Probably not something to dig into here.